Tutorial: update TOV parameter file for polytrop to be consistent with ET paper 2011

HelviWitek opened the pull request.

HelviWitek committed:
CactusTutorial: extended instructions regarding pre-requisites on a Mac

HelviWitek committed:
Revert "CactusTutorial: extended instructions regarding pre-requisites on a Mac"

This reverts commit 8049e8a2a6cf62657acd18fbd44ca0a637721ed7.

HelviWitek committed:
CactusTutorial: revert commit and fixed version of python interpreter

HelviWitek committed:
Merge branch 'master' into master

rhaas80 commented:
There seems to be no change in this pull request. Even running a diff against master:
````diff
ekohaes8: ~/websites/jupyter-et$ git diff helvi/master..origin/master
diff --git a/jupyterhub/README.md b/jupyterhub/README.md
index 69288b3..2b46baf 100644
--- a/jupyterhub/README.md
+++ b/jupyterhub/README.md
@@ -29,13 +29,6 @@ sudo ./install-kubeadm.bash
 sudo -E ./init-master.bash
 ```

-## Updating kubernetes
-
-This must not be done via `apt-get` but instead via `kubeadm upgrade`. If
-apt-get is used then errors like
-https://github.com/kubernetes/kubernetes/issues/65863 are likely to happen. Best
-to hold `kubelet`, `kubeadm` and `kubernetes-cni`.
-
 ## Updateing helm

 The 0.7 version of JupyterHub requires a newer version of helm than is installed by default. To fix run
````
shows no differences (other than a change in my own master that you don't have yet)

HelviWitek committed:
Update Cactus Tutorial

replaced example parameter file for the TOV star with that of a 2D polytrop. This is consistent with the 2011 paper and the gallery example.

HelviWitek commented:
Hi Roland,
I just finished the update and created the pull request. It would be great
if you could take a minute to check whether you agree. I updated the TOV
parameter file using a 2D polytrop. I also did *not* include checkpointing
parameters that cannot be used for the tutorial to make it shorter and more
readable.
cheers,
Helvi

On Tue, Jul 2, 2019 at 5:13 PM Roland Haas <notifications@github.com> wrote:

> There seems to be no change in this pull request. Even running a diff
> against master:
>
> ekohaes8: ~/websites/jupyter-et$ git diff helvi/master..origin/masterdiff --git a/jupyterhub/README.md b/jupyterhub/README.md
> index 69288b3..2b46baf 100644--- a/jupyterhub/README.md+++ b/jupyterhub/README.md@@ -29,13 +29,6 @@ sudo ./install-kubeadm.bash
>  sudo -E ./init-master.bash
>  ```
> -## Updating kubernetes--This must not be done via `apt-get` but instead via `kubeadm upgrade`. If-apt-get is used then errors like-https://github.com/kubernetes/kubernetes/issues/65863 are likely to happen. Best-to hold `kubelet`, `kubeadm` and `kubernetes-cni`.-
>  ## Updateing helm
>
>  The 0.7 version of JupyterHub requires a newer version of helm than is installed by default. To fix run
>
> shows no differences (other than a change in my own master that you don't
> have yet)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nds-org/jupyter-et/pull/14?email_source=notifications&email_token=AHPEURICPZRYWRL35VDSDD3P5NWB3A5CNFSM4H44E5P2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZBTNFY#issuecomment-507721367>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHPEURLMHVONO3FJHQGLEQ3P5NWB3ANCNFSM4H44E5PQ>
> .
>

HelviWitek committed:
Merge pull request #1 from HelviWitek/HelviWitek-patch-1

Update Cactus Tutorial

rhaas80 commented on CactusTutorial.ipynb:
```diff
@@ -392,11 +390,14 @@
+    "ML_BSSN::BetaDriver          = 0.0\n",  
+    "ML_BSSN::advectLapse         = 0\n",
+    "ML_BSSN::advectShift         = 0\n",
+    "ML_BSSN::MinimumLapse        = 1.0e-8\n",
```
They do match what we had in the ET paper. On the downside those
options are really only usable for TOV oscillations as a
ShiftGammaCoeff = 0. and BetaDriver = 0 will select geodesic slicing
which tends to be a bad idea for anything that is not stationary
spacetime (certainly going to blow up for a collapsing TOV star).

HelviWitek replied:
Let's add a warning that the current parameters select geodesic slicing and only work for stationary spacetimes.
I suggest to add the parameters that allow for actual collapse as comment unless it would be more complicated.

HelviWitek replied:
I added a warning for now. Once we have tested the collapse, I will add the appropriate parameters as comment.

rhaas80 commented on CactusTutorial.ipynb:
```diff
@@ -434,26 +435,11 @@
-    "EOS_Omni::poly_k                       = 100\n",
-    "EOS_Omni::gl_gamma                     = 2\n",
-    "EOS_Omni::gl_k                         = 100\n",
-    "\n",
```
I may actually opt to add back the 

"# Set equation of state for evolution\n",
"EOS_Omni::poly_gamma                   = 2\n",
"EOS_Omni::poly_k                       = 100\n",
"EOS_Omni::gl_gamma                     = 2\n",
"EOS_Omni::gl_k                         = 100\n",

options to both this and the gallery parfile (even if they just set
things to the same value as the defaults) so that these sample parfiles
do not rely on the magic of default values for parameters.

HelviWitek replied:
Agreed

HelviWitek replied:
done.

rhaas80 commented on CactusTutorial.ipynb:
```diff
@@ -368,15 +366,15 @@
+    "GRHydro::GRHydro_stencil        = 3\n",
+    "GRHydro::bound                  = \"none\"\n",
+    "GRHydro::rho_abs_min            = 1.e-10\n",
+    "#new finite difference order parameter\n",
```
would suggest against using the word "new" here since "new" will turn
into "old" with time. Instead "parameter controlling finite difference
order for Christoffel symbols in GRHydro" which is what it controls
(though I would not be surprised if some of the vector potential code
uses it as well).

HelviWitek replied:
Agreed

HelviWitek replied:
done

rhaas80 reviewed (changes requested):
Minor suggestions to change a comment and to keep the EOS parameters. Other stuff looks fine and comments are mostly a note to oneself to get collapse tov parfile working (again).
Maybe add a comment this parfile using BSSN parameters no suitable for a collapse to a BH.

HelviWitek committed:
Cactus Tutorial

updated the TOV parameter file to address the reviewer's comments.

rhaas80 commented:
I made some cosmetic changes (no merging of lines which would be wider than the others already present) and kept the onefile output option (along with its warning) since having a single output file is useful for the tutorial setting.
I am just trying to see if this all still works and if the results do indeed look visually similar to the gallery example.

rhaas80 commented:
Applied as git hash [7ebc1ed](https://github.com/nds-org/jupyter-et/commits/7ebc1ed13c4586a48f4d9329c912d8bbbac73d77) "CactusTutorial: more closely match gallery BNS example physics" of [jupyter-et](https://github.com/nds-org/jupyter-et)
e
